import_logs.py

```diff
@@ -404,7 +404,7 @@ def get(self, key):
             return '200'
         elif key == 'user_agent':
             user_agent = super(AmazonCloudFrontFormat, self).get(key)
-            return urllib2.unquote(user_agent)
+            return urllib2.unquote(urllib2.unquote(user_agent))  # Value is double quoted!
```

Comment:
According to AWS docs the useragent should only be quoted once. But guess it shouldn't hurt to unquote a second time, as it shouldn't change anything

Reply:
That's what I thought, but from practice it was quoted twice for some reason.